src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/ParametersTest.cs

```diff
@@ -520,5 +520,232 @@ private enum MyEnum
             A = 1,
             B = 2
         }
+
+        private static void ExecuteNonQueryCommand(string connectionString, string cmdText)
+        {
+            using (SqlConnection conn = new SqlConnection(connectionString))
+            using (SqlCommand cmd = conn.CreateCommand())
+            {
+                conn.Open();
+                cmd.CommandText = cmdText;
+                cmd.ExecuteNonQuery();
+            }
+        }
+
+        [Fact]
+        private static void EnableOptimizedParameterBinding_ParametersAreUsedByName()
```

Comment:
Those scenarios are already covered by the existing parameter tests. I just chose to add these new behaviours as new tests to make it clear what they were testing instead of changing the existing covering tests.